primitives/beefy/src/lib.rs

```diff
@@ -201,6 +201,9 @@ sp_api::decl_runtime_apis! {
 	/// API necessary for BEEFY voters.
 	pub trait BeefyApi
 	{
+		/// Return the block number where BEEFY consensus is enabled/started
+		fn beefy_genesis() -> Option<NumberFor<Block>>;
```

Comment:
nitpick: Maybe just `genesis`?

Reply:
I agree it is clear when reading `<T as BeefyApi>::genesis()`, but in practice runtime apis are called without trait disambiguation and usually look like:
```diff
runtime
	.runtime_api()
-	.genesis(&at)		// this is confusing at first sight
+	.beefy_genesis(&at)	// this is more explicit/clear IMO
```